rbc/tests/test_omnisci_mlpack.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import re
+from rbc.tests import omnisci_fixture
+from rbc.errors import OmnisciServerError
+
+
+@pytest.fixture(scope='module')
+def omnisci():
+
+    for o in omnisci_fixture(globals(), minimal_version=(5, 6)):
+        yield o
+
+
+@pytest.mark.parametrize("func", ['db_scan', 'kmeans'])
```

Comment:
The name is actually `dbscan`, without the underscore.

```suggestion
@pytest.mark.parametrize("func", ['dbscan', 'kmeans'])
```